PKHeX.Core/Saves/Access/SaveBlockAccessor8SWSH.cs

```diff
@@ -175,9 +175,9 @@ public sealed class SaveBlockAccessor8SWSH(SAV8SWSH sav) : SCBlockAccessor, ISav
     public const uint KCapturedBlacephalon = 0xF769AFDF;
 
     // Gift event flags (bool)
-    public const uint KReceivedGiftBulbasaur = 0x4F240749;
+    public const uint KReceivedGiftChoiceBulbasaur = 0x4F240749;
     public const uint KReceivedGiftCharmander = 0x178159E5;
```

Comment:
Any reason why Charmander was not revised?

Reply:
Charmander is not a choice, it's just sitting in Hop's house at postgame.
Bulbasaur/Squirtle are a choice in IoA.